configs/keycloak_bridge.yml

```diff
@@ -68,6 +69,7 @@ db-audit-ro-max-idle-conns: 2
 db-audit-ro-conn-max-lifetime: 10
 db-audit-ro-migration: false
 db-audit-ro-migration-version: 0.1
+db-audit-ro-connection-check: false
```

Comment:
This is just for our test environments, right ? We don't want to use this setting when deployed ?

Reply:
This is just for our local dev env